fem/dataset.py

```diff
@@ -6,8 +6,8 @@
 import torch
 from torch.utils import data
 import imageio
+import skimage.color
 import numpy
-import cv2
```

Comment:
Why do you get rid of opencv?

Reply:
i use imageio for file loading since opencv implicitly converts rgb to bgr. This introduces hard to notice bugs